### PR TITLE
12.19.1 Group ID Change

### PR DESCRIPTION
**Pull request by ab9967**
<b>Changes part of this PR</b>

- Group ID Change from 'com.cnp' to 'io.github.vantiv'
- Version ID updated to '12.19.1'
- Added New task for .pom file generation in build.gradle
- Updated existing task for JavaDocs with type:JAR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ Setup
...
     1. For Maven:
         ```xml
             <dependency>
 
         
 
-                <groupId>com.cnp</groupId>
+                <groupId>io.github.vantiv</groupId>
                 <artifactId>cnp-sdk-for-java</artifactId>
                 <version>12.0.3</version>
```
Comment (VantivSDK) [inline README.md]: should this version say 12.19.1 as well?

**Review by VantivSDK — CHANGES_REQUESTED** [review]
Update needed for changelog

File: README.md (hunk 1 above)
Comment (ab9967) [reply README.md]: Fixed

> Update needed for changelog
Fixed

**Review by VantivSDK — APPROVED** [review]

**Comment by ab9967** [comment]
Merging pull req - changes related to GroupID

**Pull request merged**